Views/PricePartSettings.Edit.cshtml

```diff
@@ -0,0 +1,51 @@
+@model PricePartSettingsViewModel
+
+<fieldset class="form-group" asp-validation-class-for="PricePartSettings">
+    <label asp-for="CurrencySelectionMode">@T["Currency selection mode"]</label>
+
+    <div class="input-group">
+        <div class="input-group-append">
+            <select asp-for="CurrencySelectionMode"
+                    asp-items="@(new SelectList(Model.CurrencySelectionModes, "Text", "Value"))"></select>
+        </div>
+    </div>
+
+    <span class="hint">@T["Specifies the currency configuration for this PricePart."]</span>
+
+    <fieldset class="input-group" id="speccurr">
```

Comment:
Avoid statically-defined ids: while it's unlikely in this particular situation, these views should be designed so there can be more than one copy on the page. I think you should be able to use a `Html.Id`.

Reply:
Thanks, great feedback. Implemented it using Html.IdFor.